src/lib.rs

```diff
@@ -198,6 +198,98 @@ impl_error!(bitcoin::hashes::hex::Error, Hex, Error);
 #[cfg(test)]
 mod test {
     use super::*;
+    use bitcoin::Amount;
+    use electrsd::{
+        bitcoind, bitcoind::bitcoincore_rpc::bitcoincore_rpc_json::AddressType,
+        bitcoind::bitcoincore_rpc::RpcApi, bitcoind::BitcoinD, ElectrsD,
+    };
+    use electrum_client::ElectrumApi;
+    use lazy_static::lazy_static;
+    use std::sync::{Mutex, Once};
+    use std::time::Duration;
+
+    lazy_static! {
+        static ref BITCOIND: BitcoinD = {
+            let bitcoind_exe =
+                bitcoind::downloaded_exe_path().expect("bitcoind version feature must be enabled");
```

Comment:
Normally we read an env variable for the bitcoind/electrsd binaries and only if that's not set we try to use the embedded one. I personally use this on nixos because the binaries downloaded by the crate don't work right out of the box, and it's just easier to have my own bitcoind/electrsd and use them

Reply:
I think they are called `BITCOIND_EXE` and `ELECTRSD_EXE`in bdk

Reply:
Done in https://github.com/bitcoindevkit/rust-esplora-client/pull/10/commits/7e61f2c44e7957afffd74b2c2f18b2fdf89274bc

Reply:
I now adopted the somewhat inconsistent `ELECTRS_EXE` naming from BDK (vs. `ELECTRSD_EXE`) to make it more convenient.